ui/apps/profile/src/components/pages/profile-info-page.tsx

```diff
@@ -79,13 +83,53 @@ const ProfileInfoPage: React.FC<RootComponentProps> = props => {
     cb();
   };
 
-  const handleEntryFlag = (itemId: string, itemType: EntityTypes, user: string) => () => {
+  const handleCopy = () => {
+    /** handle copy */
+    const profileUrl = `${location.origin}/@akashaorg/app-profile/${profileId}`;
```

Comment:
since we are in the profile-info page `/@akashaorg/app-profile...` is part of the window.location. It's `location.pathname`

Reply:
Yeap. Actually the whole profile link is accessible from location. I thought to take only the origin and compose the rest of the link.